tests/indentblockcommand.js

```diff
@@ -46,6 +57,14 @@ describe( 'IndentBlockCommand', () => {
 			command = new IndentBlockCommand( editor, indentBehavior );
 		} );
 
+		describe( 'refresh()', () => {
+			it( 'should be executed for a top-most selected block', () => {
+				setData( model, '[<parentBlock><paragraph>foo</paragraph></parentBlock>]' );
+				command.refresh();
+				sinon.assert.notCalled( indentBehavior.checkEnabled );
```

Comment:
It's better to use `command.isEnabled` to determine whether indentation is available or not.  Here you break encapsulation, because **you already know** that internally `refresh` calls `checkEnabled()`.

Reply:
I'm testing the wiring of two classes here. The `IndentBehavior` interface defines this method and the test check if it is called in a proper way.

Reply:
Althought I might change this test so the behavior always returns `true` and check if command is disabled anyway.

Reply:
Done